### DISCUSSION TITLE
Disable free text on Dashboard filters

### DISCUSSION THREAD

**Discussion by Davidkramer1999** [apache/superset#28003] [Ideas]
On dashboard where you have filers it would be nice to be able to disable users to "add" value... Enter triggers serach and if in filter there is no value it adds it... I know later when you clear the drop down  the value entered beffore disappears.  Right now i'm using v4.0.0

https://github.com/apache/superset/assets/74869455/ef732742-9735-44d0-b627-fd633aee07a0

**Comment by xavier-GitHub76** [comment]
+1

**Comment by edwardzhanged** [comment]
+1
